### PR TITLE
Add sanitize for hex color

### PR DESCRIPTION
**Pull request by dingo-d**
Added `sanitize_hex_color` and `sanitize_hex_color_no_hash` to the list of sanitizing functions.
As mentioned in #780 by paulschreiber.

### CODE REVIEW

File: WordPress/Sniff.php [hunk 1]
```diff
@@ -265,6 +265,8 @@
 		'sanitize_title'             => true,
 		'sanitize_user_field'        => true,
 		'sanitize_user'              => true,
+		'sanitize_hex_color'	     => true,
+		'sanitize_hex_color_no_hash' => true,
```
Comment (grappler) [inline WordPress/Sniff.php]: The function names are sorted alphabetically so these functions should be added to line 254.

Comment (dingo-d) [inline WordPress/Sniff.php]: Sorry, didn't notice this. Will fix it now.

File: WordPress/Sniff.php [hunk 2]
```diff
@@ -252,6 +252,8 @@
 		'sanitize_email'             => true,
 		'sanitize_file_name'         => true,
 		'sanitize_html_class'        => true,
+		'sanitize_hex_color_no_hash' => true,
```
Comment (grappler) [inline WordPress/Sniff.php]: It is a tiny thing but `ht` would come after `he`.

Comment (dingo-d) [inline WordPress/Sniff.php]: On it! I should have just pulled the array out in phptester and sort it alphabetically XD

**Pull request merged**